tap_looker/sync.py

```diff
@@ -126,11 +127,17 @@ def sync_endpoint(client, #pylint: disable=too-many-branches
 
     # Get data, API request
     body = endpoint_config.get('body')
-    data = client.request(
-        method=method,
-        path=path,
-        endpoint=stream_name,
-        json=body)
+    data = []
+    try:
+        data = client.request(
+            method=method,
+            path=path,
+            endpoint=stream_name,
+            json=body)
+    except Server5xxError as err:
+        LOGGER.warning('Server 5xx Error: {}'.format(err))
+        LOGGER.warning('Error URL: {}'.format(url))
+        return 0
```

Comment:
Why return 0 here and not just propagate / raise the exception up the stack?

Reply:
We are getting a 5xx error for one User on the `content_views` stream; it works for 50+ other Users but not this one active user who views content regularly. 
This is a child endpoint and not a critical endpoint. I have logged a bug with Looker API team, but they have acknowledged that it is a bug.
We don't want the integration to stop/fail because of this error.

Reply:
The client does an iterative backoff and 5xx retry.
If it gets back to sync w/ 5xx, it would be ignored.

This works for ALL users, except one, just by changing out the user_id.
https://bytecode.looker.com:19999/api/3.1/content_view/search?user_id=11

This **DOES NOT** work for user_id = 73.
https://bytecode.looker.com:19999/api/3.1/content_view/search?user_id=73
^^ This returns a 500-Server Error.

User w/ user_id = 73 is an Active (non-Disabled) user.

Some users who are less-active or even disable, still return results. However their results are blank:
[ ]
(empty array)

For this one User - out of 50+ Users total - I get is 500: Internal Server Error w/ this message:
```
{
    "message": "An error has occurred.",
    "documentation_url": "http://docs.looker.com/"
}
```

I guess we could remove this change to ignore this error. Or maybe only ignore this error on this endpoint, `content_views`. Then when Looker fixes this known bug, we could remove this try/catch ignore.

Reply:
I understand and discussed with the team. Given the circumstances logging and moving on is fine for now. But if they respond they have fixed the bug please let us know or submit a PR to remove the catch and logging of the 5xxs.